src/main/java/ru/yandex/practicum/filmorate/storage/FilmStorage.java

```diff
@@ -13,6 +13,4 @@ public interface FilmStorage {
 
     Film getById(Integer id);
 
-    List<Film> getTopFilms(Integer count);
```

Comment:
Предлагаю не удалять.

```suggestion
    List<Film> getTopFilms(Integer count);
```